modules/oc-mirror-about-ts-v2.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-about-ts-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 issues
+
+In oc-mirror plugin v2, there are specific behaviors to handle errors during the mirroring process, especially when issues arise with operator images, additional images, or release images. Understanding how the tool behaves in these scenarios can help users troubleshoot and resolve issues more effectively.
+
+*Key Behaviors of oc-mirror plugin v2*:
+
+* Mirroring Continuation:
+
+When errors occur while mirroring operator or additional images, oc-mirror plugin v2 continues to mirror other images instead of halting the entire process. However, if there are errors during the mirroring of release images, the process is aborted.
+
+* Cluster Resource Generation:
+
+oc-mirror plugin v2 generates cluster resources (such as IDMS and ITMS files) based on the subset of images that were successfully mirrored. Even when some images fail, the necessary resources for the correctly mirrored images are created.
+
+* Error Logging:
+
+All mirroring errors are logged in a separate log file for review. This log file provides detailed information about the errors encountered during the process.
+
+* Operator Package Association:
+
+When possible, oc-mirror v2 associates the image mirroring errors with the operator package that failed as a consequence. This helps users identify which operators are incomplete due to missing or failed images.
```

Comment:
Not sure if this is giving a clear message to the user, We wanted to say that if images of a  operator fails, oc-mirror does not mirror that entire operator. May be we could put this info  in more simpler terms like i added here , WDYT ?

Reply:
Done